.github/workflows/create-swagger-apiview.yml

```diff
@@ -0,0 +1,61 @@
+name: Create APIView
```

Comment:
While we have learned a lot about GH actions and dealing with artifacts, I'm starting to believe we should simply stick with DevOps for this work. It will keep this simpler and eliminate the need for more support being added to the APIView server.